packages/components/Button/src/FAB/__snapshots__/FAB.test.tsx.snap

```diff
@@ -116,9 +114,8 @@ exports[`Default FAB (iOS) 1`] = `
       Object {
         "alignItems": "center",
         "alignSelf": "flex-start",
-        "backgroundColor": "#0078d4",
+        "backgroundColor": undefined,
         "borderBottomWidth": undefined,
-        "borderColor": "#005a9e",
```

Comment:
Hm is the borderColor disappearing on buttons an issue?

Reply:
Hmmm I'm not sure, looking into it

Reply:
I'm not sure why the borderColor disappeared, but if I tried updating the iOS design-tokens package locally to match android, and the borderColor comes back, so I think it would be fixed after updating to 0.31.0

Reply:
It's back!